templates/pages/target-platforms/boards/up-squared.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Intel Up²
+image: https://assets.ubuntu.com/v1/1f53b707-INTEL_JOULE-LOGO.png?w=200
+description: The Up² board by Intel
+tags: intel, server, lts, desktop, iot
+distributions:
+  - Ubuntu Server 16.04 LTS
+---
+
+# Up&sup2;
+
+## Ubuntu Server
+
+We will walk you through the steps of installing Ubuntu Server on the Intel Up &sup2;. At the end of this process, you will have a fully fledged development or production environment.
+
+### Ubuntu image
+
+* [Up&sup2; - Ubuntu Server 16.04 LTS image](https://up-community.org/media/ms1-release/ubuntu-16.04.3-server-upboard-201017-rc1-amd64.iso)
+
+Download and copy the image to a USB flash drive by following the installation media instructions for: [Ubuntu](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-ubuntu), [Windows](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows) or [macOS](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos).
+
+### Installation instructions
+
+Booting the board from the USB flash drive will start the Ubuntu installer.
+
+1. Insert the USB flash drive in the Up&sup2; board
```

Comment:
I'm not sure how this works on the other pages but is it necessary to have an ordered list and an unordered list like this. It seems like it ought to be one or the other